pkg/utils/utils.go

```diff
@@ -259,9 +260,10 @@ func (watchedContainer *WatchedContainerData) SetContainerInfo(wl workloadinterf
 	checkContainers := func(containers iter.Seq2[int, v1.Container], containerStatuses []v1.ContainerStatus, containerType ContainerType) error {
 		var containersInfo []ContainerInfo
 		for i, c := range containers {
+			normalizedImageName := NormalizeImageName(c.Image) // FIXME missing tag !!!!
```

Comment:
Is this comment still relevant? @matthyx

Reply:
indeed, we can remove it